src/Bot.ts

```diff
@@ -152,4 +155,16 @@ export class Bot extends builder.UniversalBot {
             callback(null, null, 200);
         };
     }
+
+    // set incoming event to any because membersAdded is not a field in builder.IEvent
+    private getMessageReactionsHandler(bot: builder.UniversalBot): (event: any) => void {
+    return async function(event: any): Promise<void> {
+            let session = await loadSessionAsync(bot, event);
+            if (event.reactionsAdded && event.reactionsAdded[0].type && event.reactionsAdded[0].type === "like") {
```

Comment:
Nit: if (event.reactionsAdded && event.reactionsAdded[0].type === "like") is enough, the third condition implies the second.

Reply:
Done.